src/rules/validate-changelog.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  ChangelogFormattingError,
+  validateChangelog,
+} from '@metamask/auto-changelog';
+import { getErrorMessage, isErrorWithCode } from '@metamask/utils/node';
+
+import { buildRule } from './build-rule';
+import { PackageManifestSchema, RuleName } from './types';
+
+export default buildRule({
+  name: RuleName.ValidateChangelog,
```

Comment:
What are your thoughts on calling this rule `require-valid-changelog` to match `require-valid-package-manifest`?